src/commands/cmd_usage.cpp

```
@@ -0,0 +1,309 @@
+std::string ShortTime(const std::string& iso) {
+    if (iso.size() < 16 || iso[10] != 'T') {
+        return iso;
+    }
+    return iso.substr(5, 5) + " " + iso.substr(11, 5);
+}
+
+std::string Bar(long value, long peak, int width) {
+    if (peak <= 0) {
+        return std::string();
+    }
+    const int filled = static_cast<int>((static_cast<double>(value) / peak) * width + 0.5);
```

Clang (14.0.6)

[error] casting (double + 0.5) to integer leads to incorrect rounding; consider using lround (`#include` <cmath>) instead

(bugprone-incorrect-roundings,-warnings-as-errors)